src/libs/ReportUtils.ts

```diff
@@ -1270,7 +1289,7 @@ function isClosedExpenseReportWithNoExpenses(report: OnyxEntry<Report>): boolean
  */
 function isArchivedRoom(report: OnyxInputOrEntry<Report>, reportNameValuePairs?: OnyxInputOrEntry<ReportNameValuePairs>): boolean {
     if (reportNameValuePairs) {
-        return reportNameValuePairs.isArchived;
+        return !!(reportNameValuePairs?.isArchived ?? reportNameValuePairs?.private_isArchived);
```

Comment:
Also include https://github.com/Expensify/App/pull/45320 in here

Reply:
Cool, we can just use private_isArchived here since this [PR](https://github.com/Expensify/App/pull/45320/files) has been merged